webpack.playground.config.js

```diff
@@ -61,7 +61,15 @@ module.exports = {
   devtool: 'source-map',
   resolve: {
     // Add `.ts` and `.tsx` as a resolvable extension.
-    extensions: ['.ts', '.tsx', '.js']
+    extensions: ['.ts', '.tsx', '.js'],
+    alias: {
+      '@okta/okta-i18n-bundles': path.resolve(__dirname, 'src/util/Bundles.ts'),
```

Comment:
This is needed for importing `Util` from playground
https://github.com/okta/okta-signin-widget/pull/3596/files#diff-b6e6fef288e9661a8b3bb8eafb9df5e8273e1f64c276db4b6a70b1cd9499f543R14